docs/tutorial/t-enable-encryption.md

```diff
@@ -52,10 +54,10 @@ telnet <IP> 9093
 
 ### Enable TLS encrypted connection
 
-Once the Kafka cluster is enabled to use encrypted connection, client applications should be configured as well to connect to
+Once the Apache Kafka cluster is enabled to use encrypted connection, client applications should be configured as well to connect to
```

Comment:
```suggestion
Once the Charmed Apache Kafka cluster is enabled to use encrypted connection, client applications should be configured as well to connect to
```

Reply:
We should avoid the `Charmed Apache Kafka cluster` term.

Reply:
Yes, you are right! Maybe we could use "Charmed Apache Kafka is enabled to use encrypted connections", but it is not a big point. Up to you

Reply:
The whole sentence seemed a bit off to me. So I re-wrote it a bit.